packages/local_auth/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: local_auth
+description: Local authentication plugin for Flutter.
```

Comment:
Can this describe better what local authentication is? Maybe mention fingerprint, biometric, etc.

If you scan the list on https://pub.dartlang.org/flutter/packages and read the current description you'll have no idea how cool this plugin actually is. 😄